pom.xml

```diff
@@ -273,7 +273,7 @@
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
-        <version>4.11.0</version>
+        <version>4.12.0</version>
```

Comment:
mockito core 4.12.0 doesn't exist, I think that this might be an accidental search and replace that happened to hit a different library with the same version.